engines/bastion_katello/app/assets/javascripts/bastion_katello/repositories/details/views/repository-info.html

```diff
@@ -283,4 +283,54 @@ <h5 translate ng-show="repository.content_type == 'docker'">Upload Docker Image<
     </form>
   </section>
 
+  <section ng-show="repository.content_type === 'yum'">
+    <div class="divider"></div>
+    <h4 translate>Distribution Information</h4>
```

Comment:
'No distribution synced' below implies that I it exists, but I have not yet synced the distribution. However, that is not something we can know unless it has been synced?